### DISCUSSION TITLE
Will torchvision's new API effect pytorch lightning?

### DISCUSSION THREAD

**Discussion by bw4sz** [Lightning-AI/pytorch-lightning#11354] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Torchvision is prototyping a new API for calling multiple model model weights from the same architecture.

https://pytorch.org/blog/introducing-torchvision-new-multi-weight-support-api/

and looking for public comment:

https://github.com/pytorch/vision/issues/5088

I think this community should have a quick scan and see if there are any reasons why the new API may interfere with any of the PL design. In some ways it duplicates the checkpoint loading idea and is working in parallel with the data module train_dataloader design by trying to store information on transforms and other pre-processing techniques. 

Thoughts?

**Comment by carmocca** [comment]
We should be good, `torchvision` is not even a dependency of this project (just installed to run examples).

[Flash](https://github.com/PyTorchLightning/lightning-flash) might be another story though, cc @ethanwharris for awareness
